egui/src/widgets/button.rs

```diff
@@ -214,6 +214,7 @@ pub struct Checkbox<'a> {
     text: String,
     text_color: Option<Color32>,
     text_style: Option<TextStyle>,
+    left_sided: bool,
```

Comment:
```suggestion
    label_on_left: bool,
```

Let's use the same explicit name consistently